### ISSUE TITLE
Field-less Widgets

### ISSUE THREAD

**Issue by skeenp** [DMS-Aus/Roam#230]
Don't know if this is currently possible, but it would be good to be able to have a widget which is not linked to a field. Currently a config item which either:
- Does not have the field element
- Does not have a value for the field element (field:)
- Has null or none as the field element value (field: none)

Will either not show on the form or throw an error.

Sometimes it would be beneficial to have items like a check box which is not linked to a field, but on the form to control the visibility of other items or functions. Additionally you might want a first name & a surname widget which is concatenated in the background to a full name field.

**Comment by NathanW2** [comment]
Yeah this is a good idea.  Stuff like groups are also field-less widgets.  I have thought about it in the past.  Not a high priority for us at the current stage however open to pull requests or we might tackle it some time in the future.
